src/anyio/_core/_eventloop.py

```diff
@@ -24,7 +24,13 @@ def run(func: Callable[..., Coroutine[Any, Any, T_Retval]], *args,
     :param args: positional arguments to ``func``
     :param backend: name of the asynchronous event loop implementation – one of ``asyncio``,
         ``curio`` and ``trio``
-    :param backend_options: keyword arguments to call the backend ``run()`` implementation with
+    :param backend_options: keyword arguments to call the backend ``run()`` implementation with.
+        So you can look at the documentation of the run function of your backend to know the
+        keywords used
+
+        Note that for the ``asyncio`` backend there is a special ``use_uvloop`` keyword argument
```

Comment:
Hi,
I think it is because it is not mentionned anywhere in the documentation that if you have uvloop installed, anyio will use it as default for the event loop.

Reply:
> The other keyword arguments are specific to the backend used, so there is no need IMHO to mention them here.

Isn't `use_uvloop` also specific to one backend, but you're still mentioning it here?

I don't think this is the right place to mention that specific option. I'll think of another way to better document it.

Reply:
I've done this in another way now.

Reply:
ok